cmds/tfexplorer/main.go

```diff
@@ -160,6 +160,19 @@ func createServer(listen, dbName string, client *mongo.Client, seed string, foun
 			log.Error().Err(err).Send()
 		}
 	})
+	router.Path("/explorer").HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
```

Comment:
We could make a function out of this so we don't repeat the same for the /index.html page too.
Something like 
```go
func serverStatic(path string) http.Handler{
    return func(w http.ResponseWriter, req *http.Request){
		r, err := statikFS.Open(path)
		if err != nil {
			mw.Error(err, http.StatusInternalServerError)
			return
		}
		defer r.Close()
		w.WriteHeader(http.StatusOK)
		if _, err := io.Copy(w, r); err != nil {
			log.Error().Err(err).Send()
		}
	}
}
```